gandi/cli/modules/paas.py

```diff
@@ -77,16 +77,22 @@ def attach(cls, name, vhost, remote_name):
                      'application.')
```

Comment:
this comment is for the line 74/75   
I wondering if you should update (or not) the comment since it's now possible to choose the branch to push ?